app/screens/AppNavigator/BottomTabNavigator.tsx

```diff
@@ -27,44 +30,50 @@ function TabBarIcon (props: { name: VectorIconName, color: string }): JSX.Elemen
 }
 
 export function BottomTabNavigator (): JSX.Element {
+  const isNetworkDrawerOpen = useSelector((state: RootState) => state.networkDrawer.isOpen)
   return (
-    <BottomTab.Navigator
-      initialRouteName='Balances'
-      tabBarOptions={{ adaptive: false, showLabel: false }}
```

Comment:
I am going to open another PR for `showLabel: fix` to get it in first and also fix some UI consistency that was reported

Reply:
Ok copy